infrastructure/tooling/src/generate/generators/docker-compose.js

```diff
@@ -143,13 +154,18 @@ const uiConfig = [
 const allowedBackgroundJobs = ['built-in-workers/server'];
 
 exports.tasks.push({
-  title: `Generate docker-compose.yml`,
+  title: `Generate docker-compose files`,
   requires: [
     ...SERVICES.map(name => `configs-${name}`),
     ...SERVICES.map(name => `procslist-${name}`),
     'target-nginx.conf',
   ],
-  provides: ['target-docker-compose.yml'],
+  provides: [
+    'target-docker-compose.yml',
```

Comment:
combined three targets into one, as it includes now more logic around which services should be included. Easier to manage in single iteration

Reply:
Nice! Solid cleanup too